api/pkg/apps/webapp/relationshippartypicker.go

```diff
@@ -0,0 +1,34 @@
+package webapp
+
+import (
+	"encoding/json"
+	"github.com/nrc-no/core/pkg/apps/iam"
+	"net/http"
+)
+
+func (h *Server) PickParty(w http.ResponseWriter, req *http.Request){
```

Comment:
Can you rename to PickRelationshipParty?
```suggestion
func (h *Server) PickRelationshipParty(w http.ResponseWriter, req *http.Request){
```